src/components/Learn-Components/Card-Component/index.js

```diff
@@ -23,14 +25,13 @@ const CardComponent = ({ tutorial, path, courseCount}) => {
                 <p>
                   <span>New</span>
                 </p>
-              ) : null}
+              ) : <div style={{whiteSpace: "nowrap"}}>
+                <span>Coming Soon</span>
```

Comment:
Instead of having a check just add 'Coming Soon' as it is.

Reply:
Nice.